packages/sources/cfbenchmarks-test/src/endpoint/rest/crypto.ts

```diff
@@ -0,0 +1,74 @@
+import { BatchWarmingTransport } from '@chainlink/external-adapter-framework/transports'
+import { EmptyObject } from '@chainlink/external-adapter-framework/util'
+import { EndpointTypes, getIdFromBaseQuote } from '../common/crypto'
+
+interface PayloadValue {
+  value: string
+  time: number
+}
+
+interface ProviderResponse {
+  payload: PayloadValue[]
+}
+
+export type RestEndpointTypes = EndpointTypes & {
+  Provider: {
+    RequestBody: EmptyObject
+    ResponseBody: ProviderResponse
+  }
+}
+
+// NOTE: This is using the BatchWarming transport, but the actual API endpoint is not batchable
+export const transport = new BatchWarmingTransport<RestEndpointTypes>({
```

Comment:
Since this one is just like the one below but with a different endpoint, we should abstract it into a builder function, subclass, etc

Reply:
Agreed. Done.